config/docinfo-header.html

```diff
@@ -0,0 +1,377 @@
+        CZg03ylG42gtd38dD3pMEtZepgYV7SkqIyC9V/Ujz8nArmKAK7dlspGH1T7qt6BQAibtdIq7Bvi9
+        FBO3TennjU2KBRrbGcQ9K7B+HGpTTQagJ/3dI+1Zr+1oH6FsAibtdYj79cvtSip5X0wqwZKmvzO7
+        lXTlJk1LHsSlNtUgNIN6VGV2FDUYI5KBxXLtZ4oCyiZg0m6HGAPLm5X9GUr3LNJonQLRiq1qviEp
+        65/kbvZkkiFo0nE90hbCWseIE6VRPMu1oUcETNrvEGPNajxFEHEuswOMzTE9TW3x+1LNgyYmZCa5
+        59WRGJAKmrQvluHcFiyBtY8RI5htSWq5JvV1UwYy9IeASXed4o6JYHEd4O20dwLtflcW2RC+K9fX
+        y4yAFDRRR9pjzxJotk3breZLdGZKoyh7MpChfwRMuu0QY/It/bKcDtDApbvviqDJ9fV6+UT9uSap
+        jvieNssGhtBevycjuQyLJTi7igL6R8Ck+w5xkX65YzKY5QTztg4wq8GjoMnqdbn3T71OTDAmbnlj
+        EwKTN2h3jLiT+j5LPPJ0/PAzBe2BHhIwyadTjEHozdTw0v3EYCetzzdAyXNC7L5c3aD2mrAHQCMW
+        wWMTAuimXZuk/e/2Kg8NcjGr5vvabVmCA/0mYJLfQD/2NZFW3p3DNDHwFDXf78ksfUcEFy832OUT
+        9gBYm70UcFOOkE+7tq80OvWssq8dDIaASZ4dYjxJiGwTTxLas1iysGViUMR3ZBFctJTtbLNK4M8e
+        ANdvEy1JhPzatVkKhMeDg4kSadUktYs7skpgOARM8u4UY6B602C/8Ynlo/QE1cCjvO/Is8ra7tMW
+        WVLKpPpsDwB7m1xNTAL2LEmE7Nu2SVp+qA9sZ6y4mQ4AUNYwMAIm5Qz2I3Cyr0TW2vlFoORmOpKU
+        cr8jU2u7f5/oWk99cT2JicWjyjKd80RbKKsEymrbFvubRNs2USKNjRWVLQyUgEk5HeIiBVPgZH2d
+        n3Ls13ckJnlDXdt9vGm09dRXqif7qZ5Y8vhJTAQiUPLIkkQot207kXFicm+sCKyJgEl5HeIicPI3
+        A/6lTHV+g/p+DGX5xWKiaz31cvXk6MSSxyG3oxFgk2YO/WrfFkt1bA67Wp+6aawInCRgUviAv74i
+        cPKosn71PPup87ut8xvU92N64klbH+/7xES3kXZ0NoCPfZS+EzfT8q2JmgC97QcXD9h2jBMvbBMX
+        WZqb2kTgNAGTfnSK+2n96mKD2NnAi2SaJj9/SynmOr/hfjcmJ5ayPavKzySIurxpUNdYOxr1ZBFk
+        61vWyaJdvGnpDQyqbYvA8LM0TrxtnPi7yLCLoPHfUpamMgHO9CdF0E//85f/3ahfvq+vB/U1GsBH
+        jsnAy+gAdXprqT/j+uVNV0GBlB3S1GeL78R36btxo4DbMUsDu+fqdqvfgRupjizqinYx/3u23WF/
+        tz+072dd3rsd/vp97eHaxonR328M5GNHX/pjahMtYz27XkQbuq0kuOJ4fTKEDypgMpxOMQb891LH
+        2AfR0U1SxzcxcFp7nYl60suAyanPuZgQj6u8AouzNLB7ayPXbL4TJ9vQXCcXR4t6o10Elpwkj0/0
+        hzd68tGmaayoLwVWJmAy3MnwOA3+N6pynrJPT0wErMVtvo70PmByzoAxvheLP7dZv2NQ97GSJVXC
+        9+NG9Slw0mU7OkntYtSbqXYRWFMbt3GqfSslA2VyaqwoiwS4NgETFh3jKHWIt078ucsOLyaMvyw6
+        P51e63UiJoODCphc8N1YTI7/euJ7scoE+aj6tOne2xP/rH734ztzI9WLUbq+rj5lLY1X/LEn60y8
+        /jO1iUeCI0AH44JF+3Yv9YFdjRWnqX18m8aLAsZAYwRMuKhzXHSMJzvFW6cmiledOC46t+rU4L+q
+        Ph3/auKY18Bo8AGTJcrrrO+BSS2Xta2nzWQXAYW1ZyfHiCf7wlvnjA/Hp/vK6uwTfE6OExdjSG0k
+        0DoBE+CsAVAMaARMAACAwXKsMAAAAMApAiYAAAAApwiYAAAAAJwiYAIAAABwioAJAAAAwCkCJgAA
+        AACnCJgAAAAAnCJgAgAAAHCKgAkAAADAKQImAAAAAKcImAAAAACcImACAAAAcIqACQAAAMApAiYA
+        AAAApwiYAAAAAJwiYAIAAABwioAJAAAAwCkCJgAAAACnCJgAAAAAnCJgAgAAAHCKgAkAAADAKQIm
+        AAAAAKcImAAAAACcImACAAAAcIqACQAAAMApAiYAAAAApwiYAAAAAJwiYAIAAABwioAJAAAAwCkC
+        JgAAAACnCJgAAAAAnCJgAgAAAHCKgAkAAADAKQImAAAAAKcImAAAAACcImACAAAAcIqACQAAAMAp
+        AiYAAAAApwiYAAAAAJwiYAIAAABwioAJAAAAwCkCJgAAAACnCJgAAAAAnCJgAgAAAHCKgAkAAADA
+        KQImAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACs7P8LMAAzhkqr9a6OtwAAAABJRU5E
+        rkJggg==" alt="OpenCL Logo" width="50%">
```

Comment:
Left aligned is good, it matches the document titles.

I did try fiddling with the width as it's quite wide on a wide-screen monitor, but everything I tried made some configuration look worse.

Reply:
Interesting, the logo looked fine for me via Chrome, but it looked quite large via Firefox (the Vulkan spec seems to behave similarly, so this isn't OpenCL-specific).  The issue seemed to be related to `width="50%"`.  I made a slightly smaller logo instead and specified the absolute image width rather than a percentage and this seemed to work OK on both browsers.  Have a look - maybe this is better?

Reply:
Interestingly I get much the same result in both Firefox and Chromium (which surely renders the same as Chrome).

And yes, much the same for Vulkan too (logo is always 50% of the screen width as the attribute says).

The smaller logo looks good to me (in Firefox and Chromium).